doto-frontend/src/components/pages/Settings/SettingsPage.js

```diff
@@ -104,25 +107,69 @@ const WorkingHoursPicker = props => {
     );
 };
 
+const useStyles = makeStyles(theme => ({
+    blue: {
+        color: theme.palette.getContrastText(blue[500]),
+        backgroundColor: blue[500],
+        boxShadow: theme.shadows[5],
+        marginLeft: "10vw",
+    },
+}));
+
 // Using props to change the colour theme of the webpage when changed by the user
 const ThemePicker = props => {
-    const handleChangeThemeToDark = () => {
-        props.changeTheme(Themes.DARK);
-    };
+    const classes = useStyles();
+
+    const handleThemeClick = (themeColour, cost) => {
+        // @params themeColour and cost
 
-    const handleChangeThemeToLight = () => {
-        props.changeTheme(Themes.LIGHT);
+        switch (themeColour) {
+            case "blue":
+                props.changeTheme(Themes.DARK);
+                break;
+            case "green":
+                props.changeTheme(Themes.LIGHT);
+                break;
+
+            case "gray":
+                break;
+            case "magenta":
+                break;
+            case "purple":
+                break;
+            case "crimson":
+                break;
+            case "black":
+                break;
+            case "red":
+                break;
+            case "darkSeaGreen":
+                break;
+            case "antiqueWhite":
+                break;
+            case "darkKhaki":
+                break;
+            case "darkSlateBlue":
+                break;
+            default:
+                break;
```

Comment:
Are these meant to be placeholder?

Reply:
Yup, the additional themes is being done by #235, so this is what they would fill in when adding their themes.